functions/assignOfflineContact.ts

```diff
@@ -98,7 +98,7 @@ const assignToOfflineWorker = async (
   const availableActivity = await context
     .getTwilioClient()
     .taskrouter.workspaces(context.TWILIO_WORKSPACE_SID)
-    .activities.list({ friendlyName: 'Available' });
+    .activities.list({ available: 'true' });
 
```

Comment:
One additional thing -- do we want to add a log if there are more than one of these, as discussed during our standup?  Such as:
```suggestion
if (availableActivity.length > 1) {
  console.warn(`There are ${availabilityActivity.length} available worker activities, but there should only be one.`);
}
```